src/bundle/Resources/mappings/symfony-service-tag-name-map.php

```diff
@@ -0,0 +1,143 @@
+    'ezplatform.fieldtype.page.serializer.handler' => 'ibexa.field_type.page.serializer.handler',
+    'ezplatform.field_type.parameter_provider' => 'ibexa.field_type.view.parameter.provider',
+    'ezplatform.filter.criterion.query_builder' => 'ibexa.filter.criterion.query.builder',
+    'ezplatform.filter.sort_clause.query_builder' => 'ibexa.filter.sort_clause.query.builder',
+    'ezplatform.form_builder.attribute_form_type_mapper' => 'ibexa.form_builder.form.type.attribute.mapper',
+    'ezplatform.form_builder.field_mapper' => 'ibexa.form_builder.field.mapper',
+    'ezplatform.form_builder.field_submission_converter' => 'ibexa.form_builder.field.submission.converter',
+    'ezplatform.form_builder.validator_constraint_mapper' => 'ibexa.form_builder.field.constraint.validator.mapper',
+    'ezplatform.form_builder.validator_form_type_mapper' => 'ibexa.form_builder.form.type.validator.mapper',
+    'ezplatform.form.trash_location_option' => 'ibexa.admin_ui.form.trash_location_option',
+    'ezplatform_graphql.domain_schema_worker' => 'ibexa.graphql.domain.schema.worker',
+    'ezplatform_graphql.fieldtype_input_handler' => 'ibexa.graphql.field_type.input.handler',
+    'ezplatform_graphql.richtext_input_converter' => 'ibexa.graphql.richtext.input.converter',
+    'ezplatform_graphql.schema_domain_iterator' => 'ibexa.graphql.domain_schema.iterator',
+    'ezplatform.http_cache.purge_client' => 'ibexa.cache.http.purge_client',
+    'ezplatform.installer' => 'ibexa.installer',
+    'ezplatform.limitation.language.version_target_evaluator' => 'ibexa.permissions.limitation_type.language_target_evaluator.version',
+    'ezplatform.page_builder.attribute_form_type_mapper' => 'ibexa.page_builder.form_type_attribute.mapper',
+    'ezplatform.persistence.legacy.gateway.shared' => 'ibexa.storage.legacy.gateway.shared',
+    'ezplatform.persistence.site_factory.criterion_handler' => 'ibexa.site.factory.storage.legacy.criterion.handler',
+    'ezplatform.query_type' => 'ibexa.query_type',
+    'ezplatform.query_type.sort_clause_parser' => 'ibexa.query_type.sort_clause.parser',
+    'ezplatform.search.elasticsearch.query.content.aggregation_result_extractor' => 'ibexa.search.elasticsearch.query.content.aggregation.result_extractor',
+    'ezplatform.search.elasticsearch.query.content.aggregation_visitor' => 'ibexa.search.elasticsearch.query.content.aggregation.visitor',
+    'ezplatform.search.elasticsearch.query.content.criterion_visitor' => 'ibexa.search.elasticsearch.query.content.criterion.visitor',
+    'ezplatform.search.elasticsearch.query.content.facet_builder_visitor' => 'ibexa.search.elasticsearch.query.content.facet.builder.visitor',
+    'ezplatform.search.elasticsearch.query.content.sort_clause_visitor' => 'ibexa.search.elasticsearch.query.content.sort_clause.visitor',
+    'ezplatform.search.elasticsearch.query.facet_result_extractor' => 'ibexa.search.elasticsearch.query.facet.result_extractor',
```

Comment:
Why not `result.extractor` given we follow this approach for e.g. `CriterionVisitor` interface?